File: lab_insert.py
import sys
import sqlite3 as sql

def main():
	sqliteFile = 'firefly.sqlite'

	try:
		con = sql.connect(sqliteFile)
		cur = con.cursor()

	except sql.Error as e:
		print("Error %s:" % e.args[0])

	finally:
		con.close()

if __name__ == '__main__':
	sys.exit(main())

File: lab_insert.cpp
#include <iostream>
using namespace std;

#include "libsqlite.hpp"

int main()
{
    try
    {
        sqlite::sqlite db( "firefly.sqlite" );

        auto cur = db.get_statement();

    }
    catch( sqlite::exception e )
    {
        cerr << e.what() << endl;
        return 1;
    }

    return 0;
}